src/picongpu/include/simulation_defines/param/ionizationEnergies.param

```diff
@@ -20,38 +20,64 @@
+ */
 
 #pragma once
 
 /** \todo Make compatible for multiple atom species
  * - Probably do some calculation from energy to required field strength (or do that model dependent)
  * - Introduce some clever way of storage, like matrices for different charge states of the same species
- * - future: even different states of excitation should be possible */
+ * - future: even different states of excitation should be possible
+ */
 
 namespace picongpu
 {
-namespace SI
+namespace AU
 {
 
-    /* ionization energy for ground state hydrogen: 13.6 eV --> converted to Joule */
-    /* Joule = kg * m^2 / s^2 */
-    PMACC_CONST_VECTOR(float_X,1,IONIZATION_ENERGY,
-        2.179e-18
-    );
-    /* ionization field strength (E-field) for hydrogen: in Volt / meter
+    /** Ionization potentials
+     *
+     * Please follow these rules for defining ionization energies of atomic species,
+     * unless your chosen ionization model requires a different unit system than AU:
+     *     - enter ionization energies in SI units
+     *     - normalize each entry with the atomic unit (AU) of energy SI::ATOMIC_UNIT_ENERGY
+     *     - use float_X as the preferred data type
+     *
+     * example:
+     *     ionization energy for ground state hydrogen: 13.6 eV
+     *     1 Joule       = 1 kg * m^2 / s^2
+     *     1 eV          = 1.602e-19 J
      *
-     * usage:
-     * PMACC_CONST_VECTOR(float_X,<NUMBER_OF_PROTONS>,IONIZATION_EFIELD,
-     *     <value_1>,
-     *     ...
-     *     <value_Z>
-     * );
+     *     1 AU (energy) = 27.2 eV
+     *                   = 4.36e-18 J
+     *                   = 2 Rydberg
+     *                   = 2 x Hydrogen ground state binding energy
      *
-     * Do not forget to change the weight of the ion and the atomic numbers in
-     * their respective `.param` file */
-    PMACC_CONST_VECTOR(float_X,1,IONIZATION_EFIELD,
-        5.14e11
+     * Atomic units are useful for ionization models because they simplify the
+     * formulae greatly and provide intuitively understandable relations to a
+     * well-known system, i.e. the Hydrogen atom.
+     *
+     * for PMACC_CONST_VECTOR usage,
+     * @see libPMacc/include/math/ConstVector.hpp
+     * for finding ionization energies,
+     * @see http://physics.nist.gov/PhysRefData/ASD/ionEnergy.html
+     */
+
+    /* ionization energy for ground state hydrogen in atomic units */
+    PMACC_CONST_VECTOR(float_X,1,IONIZATION_ENERGY_HYDROGEN,
+        2.18e-18 / SI::ATOMIC_UNIT_ENERGY
+    );
+    /** Ionization field strengths
+     *
+     * Define field strengths for the barrier suppression ionization (BSI) model
+     * in SI units, normalized by the atomic unit (AU) for electric field strength.
+     */
+
+    /* ionization field strength (E-field) for hydrogen in atomic units
+     *     - conversion from V/m to AU (E-field) */
+    PMACC_CONST_VECTOR(float_X,1,IONIZATION_EFIELD_HYDROGEN,
+        5.14e11 / SI::ATOMIC_UNIT_EFIELD
```

Comment:
@ax3l Is this part save that the user must convert his units by it self or is it to complicated?